libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Logger/SFSDKLogger.h

```diff
@@ -144,4 +197,103 @@
  */
 - (void)log:(nonnull Class)class level:(DDLogLevel)level format:(nonnull NSString *)format, ...;
 
+/**
```

Comment:
Interesting. That would not work in Java (you can't override static methods).
For a good read on why java is that way see [this](https://stackoverflow.com/questions/2223386/why-doesnt-java-allow-overriding-of-static-methods).